bedrock/firefox/templates/firefox/whatsnew/index-lite.id.html

```diff
@@ -20,49 +20,35 @@
 {% endblock %}
 
 {% block content %}
-<main role="main">
-  <header class="c-page-header main-header">
-    <div class="mzp-l-content">
-      <div class="c-page-header-inner">
-        <img src="{{ static('img/logos/mozilla/wordmark-dark.svg') }}" alt="Mozilla" width="90" class="c-page-header-logo-moz">
-        <h1 class="c-page-header-up-to-date">{{ _('Your Firefox is up to date.') }}</h1>
-      </div>
-    </div>
-  </header>
-  <div class="lite-body">
-    <div class="lite-content mzp-l-content">
-      <div class="lite-kv">
-        <img src="{{ static('img/firefox/whatsnew/lite/phone.png') }}" alt="">
+  <main role="main">
+    <header class="c-page-header main-header">
+      <div class="mzp-l-content">
+        <div class="c-page-header-inner">
+          <img src="{{ static('img/logos/mozilla/wordmark-dark.svg') }}" alt="Mozilla" width="90" class="c-page-header-logo-moz">
+          <h1 class="c-page-header-up-to-date">{{ _('Your Firefox is up to date.') }}</h1>
+        </div>
       </div>
-      <div class="lite-copy">
-        <div class="lite-container">
+    </header>
+    <div class="lite-body">
+      <div class="lite-content mzp-l-content">
+        <div class="lite-copy">
           <div class="lite-heading">
             <img src="{{ static('img/logos/firefox/logo-lite.png') }}"  alt="" height="146" width="155">
-            <h1><strong>Firefox</strong> Lite Baru</h1>
+            {# L10n: the <strong> below is for visual formatting only - bolding the word 'Firefox' #}
+            <h1>{{ _('<strong>Firefox</strong> Lite') }}</h1>
```

Comment:
This should probably be an `h2` since "Your Firefox is up to date." is already an `h1` above.